src/middlewared/middlewared/plugins/bootenv.py

```diff
@@ -165,7 +165,7 @@ def activate(self, oid):
         try:
             subprocess.run([self.BE_TOOL, 'activate', oid], capture_output=True, text=True, check=True)
```

Comment:
We should do `stdout=subprocess.PIPE, stderr=subprocess.STDOUT` and check the return code because IIRC, zectl can dump errors to stdout or stderr.